src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -828,6 +828,10 @@ private function getPropertyPath($propertyPath): PropertyPath
      */
     public static function createCache($namespace, $defaultLifetime, $version, LoggerInterface $logger = null)
     {
+        if (null === $defaultLifetime) {
+            @trigger_error(sprintf('Passing null as "$defaultLifetime" 2nd argument of the "%s()" method is deprecated since Symfony 4.4, pass an integer instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(sprintf('Passing null as "$defaultLifetime" 2nd argument of the "%s()" method is deprecated since Symfony 4.4, pass 0 instead.', __METHOD__), E_USER_DEPRECATED);
```